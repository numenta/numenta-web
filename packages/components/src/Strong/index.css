/**
 * Numenta Web Platform and Sites source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2017 Numenta <http://numenta.com>
 */

.strong {
  composes: fw7 from 'tachyons';
}
