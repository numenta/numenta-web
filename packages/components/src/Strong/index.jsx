// Numenta Web Platform and Sites source code
// MIT License (see LICENSE.txt)
// Copyright © 2005—2017 Numenta <http://numenta.com>

import React from 'react'

import styles from './index.css'


/**
 *
 */
const Strong = ({children}) => (
  <strong className={styles.strong}>
    {children}
  </strong>
)

Strong.propTypes = {
  children: React.PropTypes.node.isRequired,
}

export default Strong
