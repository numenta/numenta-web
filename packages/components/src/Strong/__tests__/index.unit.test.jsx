import React from 'react'
import renderer from 'react-test-renderer'

import Strong from '../../Strong'


describe('Strong React component', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <Strong>
        Heavy Text
      </Strong>
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
