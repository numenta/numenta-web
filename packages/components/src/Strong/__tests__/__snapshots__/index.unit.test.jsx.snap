exports[`Strong React component Renders correctly 1`] = `
<strong
  className="strong">
  Heavy Text
</strong>
`;
